utils/splunk.py

```diff
@@ -36,8 +49,48 @@ def get_auth_tuple(self):
         return self.username, self.password
 
 
+class SavedSearches(object):
+
+    def __init__(self, saved_searches):
+        self.searches = filter(lambda ss: ss.name is not None, saved_searches)
+        self.matches = filter(lambda ss: ss.match is not None, saved_searches)
+        self.log = logging.getLogger('%s.%s' % (__name__, "SavedSearches"))
```

Comment:
Is it possible to get the agent's logger here? it will then also pick up the log level set in stackstate.conf.